src/components/KSlideout/KSlideout.vue

```diff
@@ -77,36 +103,43 @@ onUnmounted(() => {
   --KCardPaddingY: 32px;
   --KCardPaddingX: 48px;
 
-  .panel-background {
-    background: var(--black-45, color(black-45));
-    bottom: 0;
-    left: 0;
-    position: fixed;
-    right: 0;
-    top: 0;
-    z-index: 9999;
-  }
-
   .panel {
     background-color: var(--white, color(white));
+    display: flex;
+    flex-direction: column;
     height: 100vh;
     max-width: 500px;
     position: fixed;
     right: 0;
-    top: 0;
```

Comment:
question: instead of removing this style and adding inline styles, cant you just bind the prop value here?

You may need to apply the offset to the overlay background as well, not sure.

Reply:
Fixed.